composer.json

```diff
@@ -39,6 +39,8 @@
     "require": {
         "php": ">=7.1",
         "ext-json": "*",
+        "ext-dom": "*",
+        "ext-zip": "*",
```

Comment:
This extension is not required in a standard Symfony environment in production, adding the dependency here will cause a backcompatibility break for all systems that don't have the zip extension installed, this is not something we can afford.